client/src/AddFriend.js

```diff
@@ -39,29 +40,31 @@ class AddFriend extends React.Component {
       })
   }
 
+  // TODO: prevent page reload on empty input
   handleAdd (e) {
     // check for empty inputs
     this.setState({ isLoading: true })
     if (!this.state.friendFName || !this.state.friendLName || !this.state.friendEmail) {
-      return this.setState({ status: 406 })
+      return this.setState({ alertType: 'emptyField' })
     }
     axios.post('/api/friendships', this.state)
       .then(res => {
         console.log('got to axios.post')
         if (res.status >= 200 && res.status < 300) {
           this.setState(state => {
             const friendships = state.friendships.concat(state.friendFName + ' ' + state.friendLName)
-            const status = 201
-            return { friendships, status }
+            const alertType = 'created'
+            return { friendships, alertType }
```

Comment:
[nit] rather than doing `this.setState(state => { ...`, you can refactor this to use the usual `this.setState({ ... })`